src/test/ts/alien/Loader.ts

```diff
@@ -13,7 +13,8 @@ export interface Context {
 
 const getRoot = () => {
   return SelectorFind.descendant(Body.body(), '#root').getOrThunk(() => {
-    const root = Element.fromHtml('<div id="root"></div>');
+    const root = Element.fromTag('div');
```

Comment:
I did this because we don't have dom-globals as a dependency on this project so if I specified `Element.fromHtml<Node>('<div id="root"></div>');` it was causing a type error.